modules/messages/src/lib.rs

```diff
@@ -710,39 +716,43 @@ impl<T: Config<I>, I: 'static> InboundLaneStorage for RuntimeInboundLaneStorage<
 	}
 
 	fn get_or_init_data(&mut self) -> InboundLaneData<AccountIdOf<BridgedChainOf<T, I>>> {
-		match self.cached_data {
-			Some(ref data) => data.clone(),
-			None => {
-				let data: InboundLaneData<AccountIdOf<BridgedChainOf<T, I>>> =
-					InboundLanes::<T, I>::get(self.lane_id).into();
-				self.cached_data = Some(data.clone());
-				data
-			},
-		}
+		self.cached_data.clone()
```

Comment:
Nit: I would rename the function to just `data()`, considering that we're not initializing it here anymore.

Reply:
Also I don't think it needs to receive `&mut self` anymore. `&self` should be enough.